client/parties/action-creators.ts

```diff
@@ -161,6 +161,31 @@ export function sendChatMessage(partyId: string, message: string): ThunkAction {
   }
 }
 
+export function kickPlayer(partyId: string, targetId: number): ThunkAction {
+  return dispatch => {
+    const params = { partyId, targetId }
+    dispatch({
+      type: '@parties/kickFromPartyBegin',
+      payload: params,
+    })
+
+    dispatch({
+      type: '@parties/kickFromParty',
+      payload: fetch<void>(apiUrl`parties/${partyId}/${targetId}/kick`, {
```

Comment:
This URL feels fairly weird from a REST perspective, I think the options for fixing that are:

1) remove `/kick`( so this is `DELETE parties/:id/:target`). I assume this has issues with the distinguishing between leave/kick on the server and that's perhaps why you didn't do it? That seems likely solveable to me, either by using state about the client or by adding a query parameter (`?type=kick` or something)

2) Make this a POST, which is how we normally handle "command" URLs like this

Reply:
Yeah, initially I went with 1., but I didn't know how to distinguish this API route from leave. To be honest, I still don't know how to do that, besides adding a query param, which I will do here.

The reason why I didn't go with 2. is that pretty much _all_ of these routes seem like "command" URLs to me, so should probably make them all POST as well.

Reply:
I mean, all REST actions are effectively "commands" at the end of the day, I don't think I'd make *everything* a POST. The question really is, I guess: does this action map to one of the REST verbs well or not? If we're creating something or destroying something, it probably does map quite well (but it may be on some sub-entity, like how we use `/messages`). Sometimes the same action might need more description (so a kick and a leave may both be `DELETE`s but need a way to distinguish so we can return the right errors to clients or whatever)